xcell/mappers/mapper_eBOSSELG.py

```diff
@@ -0,0 +1,36 @@
+from .mapper_base import MapperSDSS
+import healpy as hp
+
+
+class MappereBOSSELG(MapperSDSS):
```

Comment:
Same comments here as I made above for `MapperBOSSCMASS`. Also, if the treatment is the same as `eBOSSQSO`, we could have a single `eBOSS` class. Let's decide that later.

Reply:
Done